round_I/study-assesment-round-I-87.md

```diff
@@ -0,0 +1,69 @@
+# STUDY ASSESSMENT - ROUND I
+
+Please replace '???' (triple question mark) in form below with respective answers.
+Note that this round doesn't assume deep dive into all datails of the paper materials.
+All the answers are supposed to be found within Abstract and Introduction sections if present.
+Otherwise in other sections in skimming mode.
+
+
+## Meta information
+
+|                                                 |                                       |
+| ---                                             |---------------------------------------|
+| 1. Assessor name                                | `Eugene Darashkevich`                 |
+| 2. Assessment date (`DD.MM.YYYY`)               | `28.05.2023`                          | 
+| 3. Study title                                  | `ParSym: Parallel Symbolic Execution` |
+| 4. Publication year in format `YYYY`, e.g. 2023 | `2010`                                |
+
+## Has the study raised any issues of Symbolic Execution?
+
+Please examine the paper on the subject of mentioning issues/difficulties/challenges/etc. of Symbolic Execution.
+
+5. (Yes/No): `Yes`
+
+If you answered 'Yes' please provide references to the location(s) within the study supporting your answer:  
+(add as many list items 5.x as required in format `page No., citation`. Remove unnecessary items)
+
+5.1 `p.1 Experimental results show that symbolic execution is
```

Comment:
@EugeneDar Do these references really answer the question on SE issues?

Reply:
@mximp I used other references, hope they will be more relevant